app.json

```diff
@@ -333,9 +333,7 @@
     "newrelic",
     "heroku-redis"
   ],
-  "buildpacks": [
-    {
+  "buildpacks": [{
```

Comment:
@alloy - what does this file do?

Reply:
Looks like it's something for Heroku pipelines: https://github.com/artsy/force/pull/2356

Reply:
Er, review apps.

Reply:
Yup. See https://artsy.slack.com/archives/C9ZHUJXEZ/p1523573626000216

Reply:
@alloy - would you mind updating https://github.com/artsy/force/pull/2356 with a quick overview of how to use this? Does this mean we can now point individual git branches to the outside world for review purposes? How does one configure the branch?